src/utils.ts

```diff
@@ -63,3 +64,48 @@ export async function findFuseOverlayfsPath(): Promise<string | undefined> {
 export function splitByNewline(s: string): string[] {
     return s.split(/\r?\n/);
 }
+
+export function getArch(): string {
+    // 'arch' should be used over 'archs', see https://github.com/redhat-actions/buildah-build/issues/60
+    const archs = core.getInput(Inputs.ARCHS);
+    const arch = core.getInput(Inputs.ARCH);
+
+    if (arch && archs) {
+        core.warning(
+            `Please use only one input of "${Inputs.ARCH}" and "${Inputs.ARCHS}". "${Inputs.ARCH}" takes precedence, `
+            + `so --arch argument will be "${arch}".`
+        );
+    }
+
+    return arch || archs;
+}
+
+export function getContainerfiles(): string[] {
+    // 'containerfile' should be used over 'dockerfile',
+    // see https://github.com/redhat-actions/buildah-build/issues/57
+    const containerfiles = getInputList(Inputs.CONTAINERFILES);
+    const dockerfiles = getInputList(Inputs.DOCKERFILES);
+
+    if (containerfiles.length !== 0 && dockerfiles.length !== 0) {
+        core.warning(
+            `Please use only one input of "${Inputs.CONTAINERFILES}" and "${Inputs.DOCKERFILES}". `
```

Comment:
Both "containerfiles" and "dockerfiles" inputs are set. Please use only one of these two inputs, as they are aliases of one another. "containerfiles" takes precedence.